modules/data.atmosphere/tests/Rcheck_reference.log

```diff
@@ -448,17 +416,14 @@ Argument items with no description in Rd object 'met.process.stage':
 Argument items with no description in Rd object 'met_temporal_downscale.Gaussian_ensemble':
   ‘in.path’ ‘in.prefix’
 
-Argument items with no description in Rd object 'prepare_narr_year':
-  ‘verbose’
-
 Argument items with no description in Rd object 'split_wind':
   ‘start_date’ ‘end_date’
 
 * checking for unstated dependencies in examples ... OK
 * checking contents of ‘data’ directory ... OK
 * checking data for non-ASCII characters ... OK
 * checking data for ASCII and uncompressed saves ... WARNING
-  
+
```

Comment:
Apparently the check script doesn't know how to handle whitespace changes, so this deletion was making the "significantly better compression..." warning reappear.

```suggestion
  
```